### ISSUE TITLE
can i concate condition latents only by this "hidden_states = torch.cat([hidden_states, condition_latents], dim=1)"

### ISSUE THREAD

**Issue by SuXuping** [Yuanshi9815/OminiControl#24]
i see a comment in code [# hidden_states = torch.cat([hidden_states, condition_latents], dim=1)].  
have you tested the results by concate the condition latents like the above way?  or what is the difference between above way and the new way in your code?

**Comment by sd0809** [comment]
The same question, looking forward to the author's response!

**Comment by Yuanshi9815** [comment]
Hi! @SuXuping @sd0809 

Currently, the module can't directly concatenate the condition latents with hidden states this way. While it's technically possible to concatenate them and train a new model with that approach, we chose a different implementation for a few key reasons:
1. Processing them separately makes it easier to implement LoRA adapters that only apply to the condition tokens.
2. This separate processing approach provides more flexibility, particularly for future support of multiple conditions.
